apps/glusterfs/brick_entry.go

```diff
@@ -274,6 +274,11 @@ func addVolumeIdInBrickEntry(tx *bolt.Tx) error {
 		}
 		for _, brick := range volumeEntry.Bricks {
 			brickEntry, err := NewBrickEntryFromId(tx, brick)
+			if err == ErrNotFound {
+				logger.Warning("Volume [%v] liks to orphaned "+
```

Comment:
Typo: liks -> links
Also, Heketi doesn't really have the habit of putting brackets around strings. Since it only ever prints IDs I don't think they aid readability. This is minor tho.